libtransmission/torrent.c

```diff
@@ -3467,6 +3467,20 @@ void tr_torrentGotBlock(tr_torrent* tor, tr_block_index_t block)
 ****
 ***/
 
+static void tr_findFileInDir(char const* name, char const* searchDir, char const** base, char const** subpath,
```

Comment:
Static functions don't need the `tr_` prefix since no one is going to call them outside this file. Unfortunately, not all the relevant functions follow this rule.

Reply:
There's also a mix of variable (and function) naming styles. Let's start small and try keeping it consistent at least in new code. I'm personally for snake_case in C code, but I know that @ckerr has been switching back and forth over the years...

Reply:
I'm happy to use whichever coding style will be accepted.

Reply:
```suggestion
static void find_file_in_dir(char const* name, char const* search_dir, char const** base, char const** subpath,
```
@ckerr any objections?